sunbird-cassandra-migration/cassandra-migration/src/main/resources/db/migration/cassandra/V1.107_cassandra.cql

```diff
@@ -0,0 +1,12 @@
+
+CREATE KEYSPACE IF NOT EXISTS sunbird_groups WITH replication = {'class':'SimpleStrategy','replication_factor':3};
+
+CREATE TABLE IF NOT EXISTS sunbird_groups.group (id text, name text, description text, activities list<frozen <map<text,text>>>, status text, membershipType text, createdBy text, createdOn timestamp, updatedBy text, updatedOn timestamp, PRIMARY KEY (id));
+
+CREATE TABLE IF NOT EXISTS sunbird_groups.user_group(userId text, groupId set<text>, PRIMARY KEY(userId));
+
+CREATE TABLE IF NOT EXISTS sunbird_groups.group_member(groupId text, role text, userId text, status text, createdBy text, createdOn timestamp , updatedBy text, updatedOn timestamp, removedBy text, removedOn timestamp, PRIMARY KEY(groupId, userId));
+
+CREATE INDEX IF NOT EXISTS idx_sunbird_groups_status ON sunbird_groups.group (status);
```

Comment:
Likely a replace problem. You perhaps wanted the index to be called idx_groups_status only.

Reply:
@indrajra : no, actually idx_groups_status index is already there on the existing table. So it won't take the same index on new table in another key space so i used idx_sunbird_groups_status.

Reply:
@indrajra : done